src/pages/components/drawer.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Drawer
+description: >
+  A Drawer is a container for supplementary content that is anchored to the edge of a page.
+package: '@zendeskgarden/react-modals'
+components:
+  - modals/src/elements/DrawerModal/DrawerModal.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>
+        To display information or actions that are supplementary to the screen’s primary content
+      </li>
+      <li>To display a list of actions that affect the screen’s content, such as filtering data</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To obscure the main content and focus the user on modal content, use a
+          [Modal](/components/modal)
```

Comment:
Does this need to be rewritten in light of the fact we are not obscuring main page content with a background? It is visible, but many users won't be able to consume it (low vision, but for sure screenreaders, which are focus jailed).

Reply:
Good point, to keep this moving I'll omit it for now and add it to the Google doc to refactor.